File: tests/fixtures/code_samples/go/sample_031.go
// Sample 31: small utility.
package samples

func Operation31(xs []int) int {
    total := 31
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure31(v int) int {
    return (v * 31) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_031.rs
// Sample 31: small utility.
pub fn operation_31(xs: &[i32]) -> i32 {
    let mut total: i32 = 31;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_31(v: i32) -> i32 {
    (v * 31) %% 7919
}

